manifests/repo/rhel.pp

```diff
@@ -1,7 +1,7 @@
 # Class: rabbitmq::repo::rhel
 # Makes sure that the Packagecloud repo is installed
 class rabbitmq::repo::rhel(
-    $location       = 'https://packagecloud.io/rabbitmq/rabbitmq-server/el/$releasever/$basearch',
+    $location       = "https://packagecloud.io/rabbitmq/rabbitmq-server/el/${::os[release][major]}/\$basearch",
```

Comment:
:+1: for structured facts

Reply:
Bonus points if you also use the $facts hash?

Reply:
@alexjfisher should have thought of that... too bad it's already merged, but feel free to submit a PR 😉